### PR TITLE
[FEATURE] payment template

### PR DESCRIPTION
**Pull request by sentientwaffle**
Depends on https://github.com/interledger/five-bells-shared/pull/45 and https://github.com/interledger/five-bells-ledger/pull/106

Fixes https://github.com/interledger/five-bells-connector/issues/51
Fixes https://github.com/interledger/five-bells-connector/issues/50

Based on https://github.com/interledger/five-bells-connector/pull/59

### CODE REVIEW

File: controllers/quote.js [hunk 1]
```diff
@@ -121,16 +122,23 @@ const UnacceptableExpiryError = require('../errors/unacceptable-expiry-error')
 /* eslint-enable */
 
 exports.get = function *() {
-  let quote = yield backend.getQuote({
-    source_ledger: this.query.source_ledger,
-    destination_ledger: this.query.destination_ledger,
-    source_amount: this.query.source_amount,
-    destination_amount: this.query.destination_amount
-  })
+  const query = new QuoteQuery(this.query)
```
Comment (lumberj) [inline controllers/quote.js]: Looks like there may be a conflict with https://github.com/interledger/five-bells-connector/pull/63/files#diff-07f5112404c3959fa178d83a407c7273R129

Comment (sentientwaffle) [inline controllers/quote.js]: It isn't a conflict; feature/dj-quote-balance is based on feature/dj-payment-template

Comment (lumberj) [inline controllers/quote.js]: Ah ok, so this should be merged first? LGTM

**Comment by sentientwaffle** [comment]
Since the other isn't done, sure.

**Pull request merged**